README.md

```diff
@@ -5,6 +5,7 @@ Mutation Testing in Smalltalk
 [![Coverage Status](https://coveralls.io/repos/github/pavel-krivanek/mutalk/badge.svg?branch=master)](https://coveralls.io/github/pavel-krivanek/mutalk?branch=master)
 [![Pharo version](https://img.shields.io/badge/Pharo-9.0-%23aac9ff.svg)](https://pharo.org/download)
 [![Pharo version](https://img.shields.io/badge/Pharo-10-%23aac9ff.svg)](https://pharo.org/download)
+[![Pharo version](https://img.shields.io/badge/Pharo-11-%23aac9ff.svg)](https://pharo.org/download)
```

Comment:
Hmm, I wonder if these badges are good.
Because they point to Pharo's download, and not to a pharo download with mutalk installed, right?

I wonder if we couldn't just remove these badges.

Reply:
To me they also indicate that MuTalk works on these versions, but I get your point.
Should we remove them ?